ignite/src/test/java/org/apache/zeppelin/ignite/IgniteSqlInterpreterTest.java

```diff
@@ -58,14 +58,14 @@ public void setUp() {
 
     IgniteConfiguration cfg = new IgniteConfiguration();
     cfg.setDiscoverySpi(discoSpi);
+    cfg.setPeerClassLoadingEnabled(true);
```

Comment:
this is set in default-ignite-jdbc.xml right? would test be able to pick up this setting from there?